### DISCUSSION TITLE
Set to translate only lines starting with '//'

### DISCUSSION THREAD

**Discussion by Vallenbo** [YiiGuxing/TranslationPlugin#6213] [Q&A]
Set to translate only lines starting with '//'

Example
Code:
ipv4Addr := net. ParseIP("192.0.2.1") // This mask corresponds to a / 24 subnet for IPv4.
ipv4Addr := net. ParseIP("192.0.2.1")// 此掩码对应于 IPv4 的 / 24 子网。---->true

this is err：
ipv4Addr ：= 网络。ParseIP（“192.0.2.1”)  // 此掩码对应于 IPv4 的 / 24 子网。

**Comment by YiiGuxing** [comment]
There are already plans for it.
